lib/kubernetes-deploy/options_helper.rb

```diff
@@ -2,11 +2,22 @@
 
 module KubernetesDeploy
   module OptionsHelper
-    def self.default_and_check_template_dir(template_dir)
+    def self.default_and_check_template_dir(template_dir, use_stdin: false)
+      if use_stdin
+        if $stdin.tty?
+          puts "Nothing to read from $stdin"
+          exit(1)
+        end
+
+        template_dir = Dir.mktmpdir("kubernetes-deploy")
+        input = $stdin.readlines.join("")
```

Comment:
Why `readlines.join` instead of `read`?
Similar question for below: Couldn't that be `File.write(File.join(template_dir, 'templates.yml.erb'), input)`? The template dir returned in this case should already be a full path.